intelmq/bots/collectors/n6/collector_stomp.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+import sys
+import os.path
+import stomp
+
+from intelmq.lib.bot import Bot
+from intelmq.lib.harmonization import DateTime
+from intelmq.lib.message import Report
+
+
+class StompListener(stomp.listener.PrintingListener):
+    """ the stomp listener gets called asynchronously for
+        every STOMP message
+    """
+    def __init__(self, n6stompcollector):
+        self.n6stomper = n6stompcollector
+
+    def on_heartbeat_timeout(self):
+        # XXX FIXME: use logger instead of print
+        # XXX FIXME: need to reconnect after timeout
+        print "lost connection!"
```

Comment:
nice catch, I forgot that one. Okay, fixed. Apart from that minor print thing: did you test it??